rllib/algorithms/algorithm_config.py

```diff
@@ -2556,6 +2560,27 @@ def offline_data(
                 `MultiAgentEpisode` not supported, yet). Note,
                 `rllib.core.columns.Columns.OBS` will also try to decompress
                 `rllib.core.columns.Columns.NEXT_OBS`.
+            materialize_data: If the raw data should be materialized in memory. This
+                boosts performance extensively, but needs enough memory to avoid an OOM.
+                Make sure that your cluster has the resources available. For very large
+                data you might want to switch to streaming mode, i.e. `False`. The
+                default is `False`. If your algorithm does not need the module in its
+                connectors (`ConnectorV2` pipeline) or all connectors are stateless
+                you might want to consider setting `materialize_mapped_data` to `True`
+                instead and set `materialize_data` to `False`. If your data does not fit
+                into memory set both configurations to `False`.
```

Comment:
```suggestion
            materialize_data: Whether the raw data should be materialized in memory. This
                boosts performance, but requires enough memory to avoid an OOM, so
                make sure that your cluster has the resources available. For very large
                data you might want to switch to streaming mode by setting this to
                `False` (default). If your algorithm does not need the RLModule in the
                Learner connector pipeline or all (learner) connectors are stateless
                you should consider setting `materialize_mapped_data` to `True`
                instead (and set `materialize_data` to `False`).
                If your data does not fit into memory or your Learner connector pipeline
                requires an RLModule or your connectors are stateful, set both
                `materialize_data` and `materialize_mapped_data` to `False`.
```